### PR TITLE
Various small UI naming tweaks

### PR DESCRIPTION
**Pull request by rmars**
- align top two tables in the service mesh page
- "All Deployments" -> "Deployments"
- reorder latency p50, p95, p99
- "Current success" -> "Success rate"

Signed-off-by: Risha Mars <mars@buoyant.io>

### CODE REVIEW

File: web/app/css/service-mesh.css [hunk 1]
```diff
@@ -25,6 +25,10 @@
   border-radius: var
 }
 
+.service-mesh-table {
+  margin-top: 18px;
+}
+
```
Comment (klingerf) [inline web/app/css/service-mesh.css]: This is a big improvement. I wonder if we want to do the same for `.incomplete-mesh-message, .complete-mesh-message`. We could add a 32px top margin to those classes to line them up with the header of the Data Plane table. This is how it looks now:

image

And with 32px padding:

image

What do you think?

File: web/app/js/components/LatencyOverview.jsx [hunk 2]
```diff
@@ -160,7 +160,7 @@ export default class MultiLineGraph extends React.Component {
     return (
       <div className="current-latency">
         {
-          _.map(["P99", "P95", "P50"], latency => {
+          _.map(["P50", "P95", "P99"], latency => {
```
Comment (klingerf) [inline web/app/js/components/LatencyOverview.jsx]: As part of this change can we also reorder the latency columns in TabbedMetricsTable:

image

Comment (rmars) [reply web/app/js/components/LatencyOverview.jsx]: oh yeah! forgot about those, thanks!

File: web/app/css/service-mesh.css (hunk 1 above)
Comment (rmars) [reply web/app/css/service-mesh.css]: yeah, that looks better to me! will do!

**Comment by rmars** [comment]
whoa good catch, those are supposed to be right aligned.

**Pull request merged**